config/crd/bases/medusa.k8ssandra.io_medusabackupschedules.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
You're running older version than previous generation and older than the one in the Makefile. This is probably one reason why there's so many weird CRD updates.

Reply:
Right, I force pushed after regenerating the CRDs with controller-gen v0.9.2.
It should look better now.